### DISCUSSION TITLE
Problem:Failed to load, please confirm the key status and correctly install the unlocking tool(入库失败，请自行确认密钥状态 和 正确安装解锁工具)

### DISCUSSION THREAD

**Discussion by ghost** [pjy612/SteamManifestCache#910] [Question / Answer]
![image](https://github.com/user-attachments/assets/4331a5e9-6769-4104-8955-9048c89a1e59)
How do i fix it? 
Message:Failed to load, please confirm the key status and correctly install the unlocking tool(入库失败，请自行确认密钥状态 和 正确安装解锁工具)
Happening when i enter this app id:2221490

**Comment by pjy612** [comment]
This means that you do not have any available keys, in which case SAI will not unlock for you.
Try your best to get a usable key and put it in config before using SAI

**Reply by pjy612** [reply]
[DecryptionKey 密钥](https://github.com/pjy612/SteamManifestCache/wiki/DecryptionKey)
